### DISCUSSION TITLE
I still cannot access safe mode

### DISCUSSION THREAD

**Discussion by hypulse** [TriliumNext/Trilium#4426] [Q&A] [answered]
Due to a widget script error, my Trilium page is failing to load.
I've tried following the safe mode script from:
https://github.com/zadam/trilium/blob/master/bin/tpl/trilium-safe-mode.sh

$ docker exec -it my-trilium-container /usr/bin/env sh
$ DIR=`dirname "$0"`
$ export TRILIUM_SAFE_MODE=1

However, I still cannot access safe mode.

**Comment by zerebos** [comment, marked as answer]
Try setting those environment variables (really just `TRILIUM_SAFE_MODE`) using docker, either in the compose file or in the command to start it up.
